### ISSUE TITLE
App name /version getting changed after reboot

### ISSUE THREAD

**Issue by ibyb007** [revanced/revanced-manager#497]
### Type

Other

### Bug description

After installing by root method the app getting changed to YouTube.tiktok_root after a reboot.. It's a huge bug whenever u reboot the app name get changed or either the version is downgraded & sometimes the app just fail to start up.

### Steps to reproduce

After installing in miui by root method just reboot.

### Android version

Miui 12.5 Android 11

### Manager version

0.39(latest)

### Target package name

com.google.android.youtube

### Target package version.

17.43.36

### Installation type

Root

### Patches selected.

Most of them except microg & custom branding.

### Device logs (exported using Manager settings).

```shell
--------- beginning of main
11-10 09:13:39.194 16991 16991 E manager.flutte: Not starting debugger since process cannot load the jdwp agent.
11-10 09:13:39.246 16991 16991 I Perf    : Connecting to perf service.
11-10 09:13:39.252 16991 16991 D NetworkSecurityConfig: No Network Security Config specified, using platform default
--------- beginning of system
11-10 09:13:39.261 16991 16991 I ContentProviderMonitor: PerfMonitor installContentProviders : before publish content providers
11-10 09:13:39.252 16991 16991 D NetworkSecurityConfig: No Network Security Config specified, using platform default
11-10 09:13:39.319 16991 16991 W manager.flutter: type=1400 audit(0.0:1784): avc: denied { read } for name="max_map_count" dev="proc" ino=202929 scontext=u:r:untrusted_app:s0:c127,c263,c512,c768 tcontext=u:object_r:proc_max_map_count:s0 tclass=file permissive=0 app=app.revanced.manager.flutter
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: QUALCOMM build                   : 42297a4, I9e9b240040
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: Build Date                       : 03/19/21
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: OpenGL ES Shader Compiler Version: EV031.32.02.07
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: Local Branch                     : 
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: Remote Branch                    : refs/tags/AU_LINUX_ANDROID_LA.UM.9.1.R1.11.00.00.604.070
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: Remote Branch                    : NONE
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: Reconstruct Branch               : NOTHING
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: Build Config                     : S P 10.0.6 AArch64
11-10 09:13:39.333 16991 16991 I AdrenoGLES-0: Driver Path                      : /vendor/lib64/egl/libGLESv2_adreno.so
11-10 09:13:39.339 16991 16991 I AdrenoGLES-0: PFP: 0x016ee190, ME: 0x00000000
11-10 09:13:39.350 16991 16991 I TetheringManager: registerTetheringEventCallback:app.revanced.manager.flutter
11-10 09:13:39.405 16991 16991 V MiuiFrameworkFactory: get AllImpl object = android.common.MiuiFrameworkFactoryImpl@c32cc6d
11-10 09:13:39.412 16991 16991 D SurfaceView: UPDATE null, mIsCastMode = false
11-10 09:13:39.424 16991 17018 I flutter : ReVanced API: Using CronetEngine + true
11-10 09:13:39.435 16991 17032 W SQLiteLog: (28) double-quoted string literal: "221F04DDCF86133E6D2E6344BA97034F"
11-10 09:13:39.435 16991 17032 W SQLiteLog: (28) double-quoted string literal: "22D1CC376C54BE531C0A569AA1492CA8"
11-10 09:13:39.452 16991 16991 I DynamiteModule: Considering local module com.google.android.gms.cronet_dynamite:0 and remote module com.google.android.gms.cronet_dynamite:1611
11-10 09:13:39.452 16991 16991 I DynamiteModule: Selected remote version of com.google.android.gms.cronet_dynamite, version >= 1611
11-10 09:13:39.452 16991 16991 V DynamiteModule: Dynamite loader version >= 2, using loadModule2NoCrashUtils
11-10 09:13:39.459 16991 16991 W System  : ClassLoader referenced unknown path: 
11-10 09:13:39.481 16991 16991 W linker  : Warning: "/data/user_de/0/com.google.android.gms/app_chimera/m/00000001/CronetDynamite.apk!/lib/arm64-v8a/libcronet.107.0.5304.36.so" unused DT entry: unknown processor-specific (type 0x70000001 arg 0x0) (ignoring)
11-10 09:13:39.485 16991 16991 I cr_CronetLibraryLoader: Cronet version: 107.0.5304.36, arch: aarch64
11-10 09:13:39.759 16991 17018 I flutter : dynamic_color: Dynamic color not detected on this device.
11-10 09:13:39.779 16991 17032 W SQLiteLog: (28) double-quoted string literal: "129C81370C8C8A20B85E5575BDF12E85"
11-10 09:13:39.779 16991 17032 W SQLiteLog: (28) double-quoted string literal: "22D1CC376C54BE531C0A569AA1492CA8"
11-10 09:13:39.799 17053 17053 W sh      : type=1400 audit(0.0:1785): avc: denied { read } for name="/" dev="sde54" ino=2 scontext=u:r:untrusted_app:s0:c127,c263,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
11-10 09:13:39.809 17053 17053 W sh      : type=1400 audit(0.0:1786): avc: denied { read } for name="/" dev="sde54" ino=2 scontext=u:r:untrusted_app:s0:c127,c263,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
11-10 09:13:39.819 17053 17053 I chatty  : uid=11919(app.revanced.manager.flutter) sh identical 3 lines
11-10 09:13:39.829 17053 17053 W sh      : type=1400 audit(0.0:1789): avc: denied { read } for name="/" dev="sde54" ino=2 scontext=u:r:untrusted_app:s0:c127,c263,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
11-10 09:13:39.839 17053 17053 W sh      : type=1400 audit(0.0:1790): avc: denied { read } for name="/" dev="sde54" ino=2 scontext=u:r:untrusted_app:s0:c127,c263,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
11-10 09:13:39.849 17053 17053 W sh      : type=1400 audit(0.0:1792): avc: denied { read } for name="/" dev="sde54" ino=2 scontext=u:r:untrusted_app:s0:c127,c263,c512,c768 tcontext=u:object_r:rootfs:s0 tclass=dir permissive=0 app=app.revanced.manager.flutter
11-10 09:13:39.868 16991 17012 I manager.flutte: ProcessProfilingInfo new_methods=174 is saved saved_to_disk=1 resolve_classes_delay=8000
11-10 09:13:40.269 16991 17032 W SQLiteLog: (28) double-quoted string literal: "484091CDFBF54C9C82180F6C7CC4C586"
11-10 09:13:40.270 16991 17032 W SQLiteLog: (28) double-quoted string literal: "EC1FE10E9DB819D8C5C547267E9050A8"
11-10 09:13:42.480 16991 16991 W MirrorManager: this model don't Support
```

### Screenshots or videos

![IMG_20221110_092704](https://user-images.githubusercontent.com/58368196/200997199-0c2c41f0-50fd-42fa-993b-f36cb163eb6a.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] The issue is related solely to the ReVanced Manager

**Comment by oSumAtrIX** [comment]
Downgrading as you described it is not possible on Android without uninstalling the app. Disable auto updates.

**Comment by deathline94** [comment]
> Downgrading as you described it is not possible on Android without uninstalling the app. Disable auto updates.

It is possible with root and disabled signature verification in package manager

**Comment by oSumAtrIX** [comment]
@ibyb007  You said:

> the version is downgraded

It is not possible as previously said.

**Comment by ibyb007** [comment]
> @deathline94 ReVanced does not disable signature verification.

But why the app name getting changed after reboot?Also failing to start up sometime.See the screen shot attached above.

**Comment by deathline94** [comment]
> @deathline94 ReVanced does not disable signature verification.

of course it doesnt , i meant with Lucky patcher or something similar and patching the package manager, it is possible to downgrade.

**Comment by deathline94** [comment]
> @deathline94 ReVanced does not disable signature verification.

Oh of course it doesn't. i meant Lucky patcher . i think i misunderstood the whole thing. Yeah its not possible that Revanced manager downgrade the Youtube app or any app for that matter . but the bug is still exists . it changes the name and of course force closes on opening . actually Twitter is the same as well . but i only had this problem with the 0.0.40 version of the manager and 0.0.39 works like a charm for me

**Comment by ibyb007** [comment]
> 

But in miui I'm facing this issue in every version of Revanced Manager.

**Comment by oSumAtrIX** [comment]
As previously said, disable auto updates for the app you mount the patched base apk file over. Same procedure as with Vanced.

**Comment by ibyb007** [comment]
> As previously said, disable auto updates for the app you mount the patched base apk file over. Same procedure as with Vanced.

Auto update is always disabled in my phone & I never faced this issue with vanced manger.

**Comment by Tiebe** [comment]
> As previously said, disable auto updates for the app you mount the patched base apk file over. Same procedure as with Vanced.

@oSumAtrIX, how about you don't close issues without understanding them, and reopen this one. There are multiple people having this issue, and it doesn't help that you are just ignoring this.

**Comment by ibyb007** [comment]
> As previously said, disable auto updates for the app you mount the patched base apk file over. Same procedure as with Vanced.

There's a major difference with vanced manager.. Vanced manger able to install the patched version over factory version (not updated) but in ReVanced u need to install the recommended version then u able to patch & install. That's what causing this issue, I reckon.

**Comment by oSumAtrIX** [comment]
@ibyb007 No, in terms of mounting it makes no difference if you first install the app, then mount or do it the other way round. If you first mount and then install the correct version, a reboot could be required (which basically is a mount after install). Vanced Manager installed the correct version before mounting the patched version.

@Tiebe If you have this issue, please include proper context instead of responding like this. Record and demonstrate the issue in full detail properly.

**Comment by Whosondephone** [comment]
I've been having some strange issues similar to this.  I never had issues with patcher 0.0.57 and YouTube 18.16.37.  After updating to patcher 1.3.6 and YouTube 18.19.35.  Initially the custom branding patch didn't work but everything else seemed to work.  Then I woke up one morning and YouTube and YouTube Music was grayed out in my launcher.  I believe there was a message saying these apps are unavailable.  

I think I tried uninstalling them then re installing and repatching.  The patching and root install said it was successful.  However loading YouTube none of the patches worked. 

Then I tried restoring a backup using TitaniumBU.  That actually worked temporarily.  Then I started to see adds again the next day.  Unfortunately I don't remember if I was restarting or not.  It almost seems like the big G has me on their list of pirates and targeted my device.  I also have automatic updates disabled in the play store.

I'm running the stock Android 8 on my LG V20 rooted.

**Comment by Whosondephone** [comment]
After much messing around my issue seemed to resolve itself after deleting everything in /data/adb/revanced. Then using the latest manager at the time 1.3.6 and the latest YouTube 18.19.35.

**Comment by Whosondephone** [comment]
OK.  Previously I only installed YouTube.  I work nights so to me it was like morning.  Yesterday evening around 4PM I installed the latest version of YouTube music from the play store.  Revanced patcher is reporting 6.08.50.  I noticed my phone rebooting all by itself for no good reason.  Probably around 10PM.  I just noticed that both YouTube and YouTube music are both grayed out an am getting the "app isn't available" tost message.  So I suppose YouTube music is the culprit.
